tests/format/angular/angular/__snapshots__/format.test.js.snap

```diff
@@ -360,6 +360,13 @@ printWidth: 80
     [target]=" a ?. b ( ) "
     [target]=" a ?. b "
     [target]=" this . a "
+    [target]=" 'string literal' "
+    [target]=" \`template literal\` "
+    [target]=" \`template
+literal\` "
+    [target]=" tag \`template literal\` "
+    [target]=" tag \`template
+literal\` "
```

Comment:
Can you move this to a separate test like https://github.com/prettier/prettier/pull/7479/files?

This file is too big.

Reply:
I see that PR calls it `expression-binding.vue`, but expression binding for Angular is kinda convered by `attributes.component.html` as well... any suggestions as to what I should call it? Maybe `single-template-literal-value.component.html` to make it specific to this behaviour? 🤔